roles/pulp_database_config/tasks/main.yml

```diff
@@ -36,7 +36,7 @@
       no_log: true
       when: pulp_default_admin_password is defined and migrate_auth.changed
 
-  run_once: true
+  run_once: "{{ __pulp_database_config_run_once }}"
```

Comment:
This feels like a hack. And it does not support the installation of multiple clusters in the same go. Do you think we can limit the runs by running the tasks per database host delegated to the corresponding first api host?

https://docs.ansible.com/ansible/latest/user_guide/playbooks_delegation.html#delegation

Reply:
@mdellweg hopefully, and that seems like a better solution, but:

1. Right now, CI is broken and I have urgent work to do. I want to address that later.
2. It seems like it requires the roles to have awareness of the inventory (host groups?), which we currently don't do as an upstream set of roles for users to consume.
3. It would require role refactoring again, which is moderately time consuming.